openpype/settings/defaults/project_settings/maya.json

```diff
@@ -148,12 +164,13 @@
         },
         "ValidateModelName": {
             "enabled": false,
+            "database": true,
             "material_file": {
                 "windows": "",
                 "darwin": "",
                 "linux": ""
             },
-            "regex": "(.*)_(\\\\d)*_(.*)_(GEO)"
+            "regex": "(.*)_(\\d)*_(?P<shader>.*)_(GEO)"
```

Comment:
Should we enable this option by default?

Reply:
I don't think so - it is rather restrictive and no-one is using it like this except one client. It will work only if you put an effort to create that shader name database and that could be really tedious and for some type of project impossible.